clientv3/op.go

```diff
@@ -36,10 +37,15 @@ type Op struct {
 
 	// for range
 	limit        int64
-	rev          int64
 	sort         *SortOption
 	serializable bool
 
+	// for range, watch
+	rev int64
+
+	// for watch
+	prefix string
```

Comment:
I think this should use `end` instead of having a separate prefix field; watch can check the op for `if end == nil || deepEqual(end, expectedPrefix)` and throw an error if it's something else.